src/components/Calculator.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import '../index.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Calculator extends React.Component {
+  render() {
+    return (
+      <div className="calculator-grid">
+        <div className="output">
+          <div className="operand">0</div>
+        </div>
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">%</button>
+        <button type="button" className="sign">÷</button>
+        <button type="button">7</button>
+        <button type="button">8</button>
+        <button type="button">9</button>
+        <button type="button" className="sign">x</button>
+        <button type="button">4</button>
+        <button type="button">5</button>
+        <button type="button">6</button>
+        <button type="button" className="sign">-</button>
+        <button type="button">1</button>
+        <button type="button">2</button>
+        <button type="button">3</button>
+        <button type="button" className="sign">+</button>
+        <button type="button" className="span_two">0</button>
+        <button type="button">.</button>
```

Comment:
- [ ] Kindly make sure to follow the design as per the requirements which is displayed on the left.

<img width="1595" alt="Screenshot 2022-10-26 at 7 56 05 PM" src="https://user-images.githubusercontent.com/70269507/198088423-a36a2868-a25a-4144-8369-04c161f1e884.png">

Reply:
Alright. Thanks